src/models/lineChart.js

```diff
@@ -251,7 +254,7 @@ nv.models.lineChart = function() {
                     .position({left: e.mouseX + margin.left, top: e.mouseY + margin.top})
                     .chartContainer(that.parentNode)
                     .valueFormatter(function(d,i) {
-                        return yAxis.tickFormat()(d);
+                        return d == null ? "N/A" : yAxis.tickFormat()(d);
```

Comment:
The change to the line chart example wasn't me, I can back that out.

In my opinion modifying the `yAxis.tickFormat` is a hack - there should be no y-axis tick if y is null, so why would user expect to handle the null value there? It just so happens that the `yAxis.tickFormat` is used by the tooltip - so yes, it is a way for the user to handle custom formatting of null y-values in tooltip. But it means they have to apply this custom formatting to every chart line chart with null y values. And null y-values are advertised as a way to make a line chart discontinuous, so I think its better to just handle it in the tooltip.